### PR TITLE
EDGCMNSPR-3 Add swagger endpoints to the list of excluded

### PR DESCRIPTION
**Pull request by andrii-novytskyi**
https://issues.folio.org/browse/EDGCMNSPR-3
## Purpose
Currently, any module that uses edge-common-spring will fail to open swagger docs because swagger endpoints are not listed as excluded if EdgeSecurityFilter.

## Approach
Added swagger endpoints to the list of excluded in EdgeSecurityFilter by default.
Unit tests are updated.

## Pre-Merge Checklist:
 Before merging this PR, please go through the following list and take appropriate actions.

 - Does this PR meet or exceed the expected quality standards?
   - [x] Code coverage on new code is 80% or greater
   - [x] Duplications on new code is 3% or less
   - [x] There are no major code smells or security issues
 - Does this introduce breaking changes?
   - [ ] Were any API paths or methods changed, added or removed?
   - [ ] Were there any schema changes?
   - [ ] Did any of the interface versions change?
   - [ ] Were permissions changed, added, or removed?
   - [ ] Are there new interface dependencies?
   - [x] There are no breaking changes in this PR.

 If there are breaking changes, please **STOP** and consider the following:

 - What other modules will these changes impact?
 - Do JIRAs exist to update the impacted modules?
   - [ ] If not, please create them
   - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
   - [ ] Do they have all they appropriate links to blocked/related issues?
 - Are the JIRAs under active development?
   - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
 - Do PRs exist for these changes?
   - [ ] If so, have they been approved?

 Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.

 While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ All spring way edge modules, that will use this library, should add it as a depe
 After that, all requests that will call the edge module will go through the request filter from the library. 
 As a result, incoming request will contain ***x-okapi-token*** in the header, and the edge module will be able to invoke folio modules.
 
-By default, filter will make authorization and receive the ***x-okapi-token*** header only if the requested URL does not start with: /admin/health, admin/info, /_/tenant.
+By default, filter will make authorization and receive the ***x-okapi-token*** header only if the requested URL does not start with: /admin/health, admin/info, /swagger-resources, /v2/api-docs, /swagger-ui, /_/tenant.
```
Comment (craigmcnally) [inline README.md:21]: I'm confused by this section of the README.  Edge APIs should require an api key to be provided by a configurable set of sources, e.g. query argument, path parameter, or header.  If you have an X-Okapi-Token there is no need to use an edge API.

Comment (andrii-novytskyi) [reply README.md:21]: this part of readme means that filter will work and get okapi-token for all request except those that listed in excludeList

Comment (craigmcnally) [reply README.md:21]: > incoming request will contain ***x-okapi-token***
The incoming request should provide an API key, not an X-Okapi-Token.

**Review by craigmcnally — APPROVED** [review]
I'm not going to hold this up over wording in the README.  Let's get this merged and we can discuss and open a separate PR to fix the wording.

**Review by illia-borysenko-ebsco — APPROVED** [review]

**Review by illia-borysenko — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (andrii-novytskyi) [reply README.md]: updated

**Pull request merged**